test/unit/test_serialize_debug.c

```diff
@@ -39,16 +39,16 @@ bool test_debug_serialize_save() {
 	RzDebug *debug = core->dbg;
 	mu_assert_notnull(debug, "debug null");
 
-	RzBreakpointItem *bp_item = rz_debug_bp_add(debug, 0x1337, 0, false, 1, "hax", 42);
+	RzBreakpointItem *bp_item = rz_debug_bp_add(debug, 0x1337, 0, false, 1, strdup("hax"), 42);
 	mu_assert_notnull(bp_item, "bp_item null");
-	bp_item->cond = "bp_cond";
-	bp_item->data = "bp_data";
+	bp_item->cond = strdup("bp_cond");
+	bp_item->data = strdup("bp_data");
 	bp_item->delta = 2;
 	bp_item->enabled = 3;
 	bp_item->expr = "bp_expr";
```

Comment:
This one too

Reply:
Should I create a function for setting these string members?

Reply:
I created a new function. Please have a look. Thank you.